app/src/main/java/org/simple/clinic/drugs/selection/EditMedicinesScreen.kt

```diff
@@ -131,6 +131,11 @@ class EditMedicinesScreen(context: Context, attrs: AttributeSet) : LinearLayout(
 
   private fun refillMedicineClicks() = RxView.clicks(refillMedicineButton).map { PresribedDrugsRefillClicked }
 
+  override fun showRefillMedicineButton() {
+    refillMedicineButton.visibility = VISIBLE
+    doneButton.visibility = GONE
```

Comment:
Same comment as above, for hiding done button, let's make a separate method called `hideDoneButton`